webapp/models.py

```diff
@@ -46,18 +46,27 @@ class Token(DateTimeMixin):
 )
 
 
+class Author(Base):
+    __tablename__ = "author"
+
+    first_name = Column(String, nullable=False, primary_key=True)
+    last_name = Column(String, nullable=False, primary_key=True)
+    email = Column(String, nullable=False, unique=True, primary_key=True)
+
+
 class Asset(DateTimeMixin):
     __tablename__ = "asset"
 
     id = Column(Integer, primary_key=True)
     asset_type = Column(String, nullable=True)
     name = Column(String, nullable=True)
```

Comment:
Has this been replaced by 'title'?

Reply:
And removed here

Reply:
Could you please elaborate this more, I didn't quite get it.
I thought I added the migrations

Reply:
You haven't referenced `name` anywhere else in this PR. It seems you have changed it to `title`. We should try to align all our naming.